spec/ios_git_helper_spec.rb

```diff
@@ -0,0 +1,21 @@
+require_relative './spec_helper'
+
+describe Fastlane::Helper::Ios::GitHelper do
```

Comment:
Rubocop is telling me:

```
warning| RSpec/FilePath: Spec path should end with `fastlane/helper/ios/git_helper*_spec.rb`.
```

Something I'm not sure is why it's saying is for this spec but not for, say, `git_helper_spec.rb`. 🤔 

Anyways, if we were to change this, it would be appropriate to change all the others, too. Or, we can just add it to the excluded rules list. I'm more leaning towards the first option. Keen to hear what you think. Regardless, that should be a dedicated PR.

Reply:
This is because it expects the module tree declared in this file to match the path subdirectories structure; so if the file contains declarations for `module Foo` which itself contains `module Bar` itself containing `class Baz` then it expects the file to be in `Foo/bar/baz.rb`

Indeed this is not the case neither for this file but note for others, especially in spec we have many similar violations / similar specs for which we don't fully respect that convention. I think most of them have been ignored in rubocop-todo.yml for now because that's part of the many things we'd need to refactor a bit to fix all the violations and I haven't had time to fix them all (for this one I agree we'd need to align all the files to that convention once we decide to un-ignore the rule)

So bottom line I disabled that rule for now somewhere in the rubocop config (maybe only for a subset of files tho) and we'll keep that for a calmer season.

Reply:
> I think most of them have been ignored in `rubocop-todo.yml` for now

Oh, right! I didn't think of looking in there 👍 

> but what do we say to rubocop spec violations…

🤣 (_still waiting for the last book!!!_)